### PR TITLE
Table: Allow to specify the empty element

### PR DESCRIPTION
**Pull request by kobim**
Will allow to pass a JSX element in addition to a string to `emptyText`
Screen Shot 2019-07-30 at 13 49 16

### CODE REVIEW

**Comment by kobim** [comment]
@kangax our issue is with the type of that placeholder (which can be only `string` with pre-defined style)
I've committed a change that might be more suitable - being able to pass `React$Node` and insert an Element with whatever style we want. (also had to give some love to flow and eslint in the process)

File: src/components/Table/Table.jsx [hunk 1]
```diff
@@ -41,9 +41,7 @@ class Table extends React.Component<Props, State> {
     empty: false,
     emptyText: '',
     loading: false,
-    stickAt: null,
     headerData: [],
-    sort: {},
```
Comment (kangax) [inline src/components/Table/Table.jsx]: are these default values not used?

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import style from './style.scss';
 type Props = {|
   clickable?: boolean,
   empty: boolean,
-  emptyText: string,
+  emptyText: React$Node,
```
Comment (kangax) [inline src/components/Table/Table.jsx]: Much better, thanks! `React$Node` also already includes a string so this just expands the type and shouldn't affect public API in any way.

File: src/components/Table/Table.jsx (hunk 1 above)
Comment (kobim) [reply src/components/Table/Table.jsx]: it was always protected by `sort && ...`, and it caused flow issues with the tests (where it was annotated as a required attribute)

**Pull request merged**